CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,100 @@
+# Contributor and Maintainer Code of Conduct
```

Comment:
I think we were thinking of going with the NumFOCUS one, right? This is definitely a project-wide decision the SGB is supposed to be handling, but I agree that we should do that ASAP and maybe a good way to do that is to start from the bottom up, haha.

Reply:
Agree we need consensus on this! I'm fine using whichever, mostly wanted to get the ball rolling.

Reply:
Doesn't need consensus, just SGB action I think.